app/views/courses/fees.html.erb

```diff
@@ -14,7 +14,7 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= form_for course, url: publish_provider_course_path(course.provider.provider_code, course.course_code), method: :post do |f| %>
+    <%= form_for course, url: provider_course_path(course.provider.provider_code, course.course_code) do |f| %>
```

Comment:
If we making this change should we change `form_for` to `form_with`?

https://m.patrikonrails.com/rails-5-1s-form-with-vs-old-form-helpers-3a5f72a8c78a